bigbluebutton-html5/imports/startup/client/base.jsx

```diff
@@ -219,15 +219,31 @@ const BaseContainer = withTracker(() => {
   let breakoutRoomSubscriptionHandler;
   let meetingModeratorSubscriptionHandler;
 
-  const User = Users.findOne({ intId: credentials.requesterUserId });
-  const meeting = Meetings.findOne({ meetingId });
+  const options = {
```

Comment:
Should probably be named `fields` instead because there are more options than just that.